containers/daap-get-glue-metadata/config.json

```
@@ -0,0 +1,11 @@
+{
+    "name": "daap-get-glue-metadata",
+    "version": "1.0.0",
+    "registry": "ecr",
+    "ecr": {
+        "role": "arn:aws:iam::013433889002:role/data-platform-gha",
+        "account": "374269020027",
+        "region": "eu-west-2",
+        "repository": "data_platform_get_glue_metadata_ecr_repo"
```

```suggestion
        "repository": "data-platform-get-glue-metadata-lambda-ecr-repo"
```
repository is constructed from `"${var.app_name}-ecr-repo"` ([source](https://github.com/ministryofjustice/modernisation-platform/blob/main/terraform/modules/app-ecr-repo/main.tf#L4))